cg/apps/demultiplex/sample_sheet/index.py

```diff
@@ -73,35 +61,6 @@ def get_index_pair(sample: FlowCellSample) -> tuple[str, str]:
     return sample.index.replace("NNNNNNNNN", ""), sample.index2
 
 
-def is_reverse_complement_needed(run_parameters: RunParameters) -> bool:
```

Comment:
One boolean cannot be used to differentiate all decisions around this. Hence I removed it